docs/7_klasser.adoc

```diff
@@ -0,0 +1,408 @@
+== Klasser
+
+=== Obligatoriske klasser i katalogdelen
+
+==== Aktør [[klasse-aktor]]
+
+Det anbefales at URI til Aktør (foaf:Agent) er identisk med den URI som Enhetsregisteret (Brønnøysundregistrene) har på enheten/virksomheten dersom den finnes i registeret.
+
+[cols="30s,70"]
+|===
+|Engelsk navn|Agent
+|Beskrivelse|Brukes til å representere en aktør som er assosiert med modellkatalogen og/eller modellene. Dersom aktøren er en organisasjon er det anbefalt å bruke «Organization Ontology».
+|URI|foaf:Agent
+|Referanse|http://http://xmlns.com/foaf/spec/#term_Agent[http://xmlns.com/foaf/spec/#term_Agent]
+|Status|Obligatorisk
+|===
+
+==== Katalogisert ressurs [[klasse-katalogisert-ressurs]]
+
+[cols="30s,70"]
+|===
+|Engelsk navn|Catalogued Resource
+|Beskrivelse|Brukes til å representere en ressurs som er beskrevet i en katalog.
+|URI|dcat:Resource
+|Referanse|https://www.w3.org/TR/vocab-dcat-2/#Class:Resource[https://www.w3.org/TR/vocab-dcat-2/#Class:Resource]
+|Status|Obligatorisk
+|===
+
+==== Begrep [[klasse-begrep]]
+
+[cols="30s,70"]
+|===
+|Engelsk navn|Concept
+|Beskrivelse|Brukes til å representere en mental forestilling om et konkret eller abstrakt fenomen i den virkelige verden.
```

Comment:
--> Brukes til å representere begrep som er viktig for å forstå eller tolke en informasjonsmodell eller et modellelement.